### PR TITLE
Workflow healthcheck fix

### PR DESCRIPTION
**Pull request by marcauberer**
Workflow healthcheck fix, although it is not the best approach at the moment.

### CODE REVIEW

**Review by Niklas-23 — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -65,9 +65,9 @@ jobs:
 
       - name: Test Docker Container listing
         run: docker ps
-      
-      - name: API healthcheck
-        run: curl -sSLf --retry-delay 5 --retry 5 --retry-connrefused --insecure http://example.org > /dev/null
```
Comment (Splines) [inline .github/workflows/ci.yml:70]: The problem here was the wrong url `http://example.org`, right? Did you try this with `localhost:8080/actuator/health`?

Comment (marcauberer) [reply .github/workflows/ci.yml:70]: Sure, but long story short: It did not work either. I'd suggest to merge this as a temporary solution.

Comment (Splines) [reply .github/workflows/ci.yml:70]: ok

**Review by Splines — APPROVED** [review]

**Comment by Splines** [comment]
approved as a temporary solution

**Pull request merged**